src/components/OptionRow.js

```diff
@@ -163,6 +166,7 @@ function OptionRow(props) {
             <Hoverable>
                 {(hovered) => (
                     <PressableWithFeedback
+                        testID={props.keyForList}
```

Comment:
Just noticing that we probably shouldn't have used `testID` here since that's pretty much specifically used for testing, why didn't we use `id` or `nativeID`?

Reply:
FYI we updated this in https://github.com/Expensify/App/pull/33667